### PR TITLE
pre-ai updates

### PR DESCRIPTION
**Pull request by KittenMoon**
have fun

### CODE REVIEW

File: textExtra.js [hunk 1]
```diff
@@ -354,6 +357,19 @@ function TextCommandBackup(bot, message, sentvalid, msgContent, g)
...
+					var mess = message.content.split(' ').slice(3, ).join(' '); //get the name for the role
+					
+					if (delay == null || delay.trim() == "") 
+					{
+						message.author.send("`Invalid Delay`");
+						return;
+					}
+
+					reverseDelay(message, hiddenChan, mess, delay);
+				}
```
Comment (acramernc) [inline textExtra.js:372]: Can't wait to see what this does 😢

**Review by acramernc — APPROVED** [review]
Looks good, idk what this new delay thing sami added does, it's probably malicious but he made it so confusing to follow i'm not going to try

File: textExtra.js (hunk 1 above)
Comment (KittenMoon) [reply textExtra.js:372]: a little bit of trolling

File: HelperFunctions/slashFridayHelpers.js [hunk 2]
```diff
@@ -205,7 +205,15 @@ function generateOps(opsArray, authorID, prefix)
...
+	g.members.fetch(uid).then(member => {
+		member.roles.remove(babadata.countrole, "you are free to count!");
+	});
+}
+
 module.exports = {
     funnyDOWText,
-    funnyFrogText
+    funnyFrogText,
+	removeCountRuin
```
Comment (acramernc) [inline HelperFunctions/slashFridayHelpers.js:218]: Fix indentation

**Comment by acramernc** [comment]
Cout ruiner, i hardly know her

looks good 🛥️

**Comment by acramernc** [comment]
Bug fixes looks good

**Comment by Aconka** [comment]
horse

**Pull request merged**